manifests/01-operator-config.yaml

```diff
@@ -19,3 +19,7 @@ spec:
     - name: GettingStartedBanner
       visibility:
         state: Enabled
+    perspectives:
```

Comment:
Lets add comment why this perspective is disabled. Maybe even with link to the epic.